CNV- 58844: Added a <shortdesc> to Support section (#102799)
openshift/openshift-docs: modules/virt-must-gather-options.adoc

[id="parameters"]
== Parameters

[role="_abstract"]

> This document provides information about requesting assistance from Red Hat Support. You will learn about monitoring the health of your clusters and VMs, along with collecting data for Red Hat Support.

Reviewer: ```suggestion
For details on monitoring the health of your clusters and VMs, along with collecting data for Red Hat Support, see below.
```
